source/docs/enterprise/vcs/bitbucket.html.md

```diff
@@ -13,46 +13,49 @@ Additionally, plans are run when a pull request is created or updated. Terraform
 Enterprise will update the pull request with the result of the Terraform plan
 providing quick feedback on proposed changes.
 
-## Registering an OAuth Application & Client
+## Setup an Organization to use Bitbucket Cloud
 
-### Creating and Updating a Bitbucket Cloud OAuth Application
+### Create a Bitbucket Cloud OAuth Consumer
 
-You will need to register Terraform Enterprise as an OAuth Application within your Bitbucket Cloud account. Proceed to https://bitbucket.org/account/user/your-username/oauth-consumers/new. Fill out the required information and set the Redirect URI to a placeholder (ie: http://example.com), as you will need to register the Bitbucket Client with Terraform Enterprise prior to receiving this value. Check all of the permission fields that apply to you, and click Save
+You need to register Terraform Enterprise as an OAuth consumer within your Bitbucket Cloud account. Proceed to https://bitbucket.org/account/user/your-username/oauth-consumers/new. Fill out the form with the following information.
```

Comment:
can we make this into a link for folks to click on? even if the full URL is the text, having a clickable link is 💯